tests/test_formula_printer.py

```diff
@@ -336,6 +322,116 @@
             groupby=[Column("transaction")],
         ),
         '((sum(foo){tag2:"tag_value2" AND (tag:"tag_value" OR tag:"tag_valueor")} / sum(bar)){tag3:"tag_value3"} * sum(pop)) by (transaction)',
+        id="test complex nested terms with OR condition",
+    ),
+    pytest.param(
```

Comment:
Can you add a test somewhere (doesn't have to be this file) for a function on a quantile? e.g. `apdex(quantiles(0.5)(...`. Just to ensure that works?